features/bevragen/persoon-beperkt/rni/dev/overzicht-gba.feature

```diff
@@ -0,0 +1,218 @@
+    | deelnemer.omschrijving | Belastingdienst (inzake heffingen en toeslagen) |
+    | omschrijvingVerdrag    | Belastingverdrag tussen België en Nederland     |
+    | categorie              | Persoon                                         |
+    En heeft de persoon een 'rni' met de volgende gegevens
+    | naam                   | waarde                                            |
+    | deelnemer.code         | 0201                                              |
+    | deelnemer.omschrijving | Sociale Verzekeringsbank (inzake AOW, Anw en AKW) |
+    | omschrijvingVerdrag    | Artikel 45 EU-Werkingsverdrag (VWEU)              |
+    | categorie              | Verblijfplaats                                    |
+
+    Voorbeelden:
+    | fields                                                                                            |
+    | adressering                                                                                       |
+    | naam,adressering.adresregel1                                                                      |
+    | naam.voornamen,naam.geslachtsnaam,adressering.land.omschrijving                                   |
+    | adressering.aanhef,adressering.adresregel2                                                        |
+    | adressering.aanschrijfwijze.naam,adressering.adresregel1,adressering.adresregel2,adressering.land |
+    
+  Abstract Scenario: persoon heeft RNI-deelnemer gegevens voor verblijfplaats, maar er worden geen verblijfplaats velden gevraagd
+    Gegeven de persoon met burgerservicenummer '000000036' heeft de volgende gegevens
+    | naam                  | waarde   |
+    | voornamen (02.10)     | Peter    |
+    | geslachtsnaam (02.40) | Jansen   |
+    | geboortedatum (03.10) | 19830526 |
+    En de persoon heeft de volgende 'verblijfplaats' gegevens
+    | naam                         | waarde                               |
+    | land (13.10)                 | 5010                                 |
+    | rni-deelnemer (88.10)        | 0201                                 |
+    | omschrijving verdrag (88.20) | Artikel 45 EU-Werkingsverdrag (VWEU) |
+    Als gba personen wordt gezocht met de volgende parameters
+    | naam          | waarde                              |
+    | type          | ZoekMetGeslachtsnaamEnGeboortedatum |
+    | geslachtsnaam | Jansen                              |
+    | geboortedatum | 1983-05-26                          |
+    | fields        | <fields>                            |
+    Dan heeft de response een persoon met de volgende 'naam' gegevens
+    | naam          | waarde |
+    | voornamen     | Peter  |
+    | geslachtsnaam | Jansen |
+
+    Voorbeelden:
+    | fields   |
+    | naam     |
+    | naam,rni |
+
+Rule: vragen van een rni gegevensgroep veld of één of meerdere velden van een rni gegevensgroep veld met de fields parameter worden genegeerd
+
+  Abstract Scenario: <sub titel> wordt gevraagd en hele categorie persoon is in onderzoek
```

Comment:
in de titel staat 'in onderzoek', maar ik zie geen 'in onderzoek' gegevens in de Given

Reply:
dat was inderdaad fout, gaat over RNI. Zal het wel ergens vandaag gekopiëerd hebben... Titel nu aangepast